ui/src/pages/deployments/deployments.tsx

```diff
@@ -19,6 +19,7 @@ export const Deployments = () => {
     items: deployments,
     defaultSort: { field: 'name', order: 'desc' },
   })
+  const canCreate = deployments?.some((deployment) => deployment.canCreate)
```

Comment:
This logic ended up a bit strange, for example what if you have no deployments? Can we put it on collection level you think? :)

Reply:
Ah yes, the create permission makes way more sense on the collection level! I will do that